src/s1cmp.hbs

```diff
@@ -11,8 +11,6 @@
 	<!--// INLINE
```

Comment:
you might want this instead of the <script loader.js>, would be a littler bit faster